x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_form.tsx

```diff
@@ -148,6 +167,23 @@ export const DetailsStepForm: FC<CreateAnalyticsStepProps> = ({
           data-test-subj="mlAnalyticsCreateJobFlyoutJobIdInput"
         />
       </EuiFormRow>
+      <EuiFormRow
+        fullWidth
+        helpText={
+          destIndexSameAsId === true && destinationIndexNameExists && indexNameExistsMessage
+        }
+      >
+        <EuiSwitch
+          disabled={isJobCreated}
+          name="mlDataFrameAnalyticsDestIndexSameAsId"
+          label={i18n.translate('xpack.ml.dataframe.analytics.create.DestIndexSameAsIdLabel', {
+            defaultMessage: 'Destination index same as job ID',
```

Comment:
What do you think about moving the switch down (below job description) so it is directly above the Destination index form row that appears if you set the switch to `off`, as these two are directly connected.

Reply:
Makes sense - updated in 30167bcdc8fc56f07afc40d00b5dfd3ef64190c0